lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -686,7 +704,7 @@ deleteTxMetas wid = deleteWhere [ TxMetaWalletId ==. wid ]
 -- | Add new TxMeta rows, overwriting existing ones.
 putTxMetas :: [TxMeta] -> SqlPersistT IO ()
 putTxMetas metas = dbChunked repsertMany
-    [(TxMetaKey txMetaTxId txMetaWalletId, m) | m@TxMeta{..} <- metas]
+    [(TxMetaKey txMetaTxId txMetaWalletId txMetaSlotId, m) | m@TxMeta{..} <- metas]
```

Comment:
This looks good.
There also needs to be a change to `selectTxHistory` (in this PR or the next) so that Pending txs are filtered out if there is a subsequent InChain tx.

Reply:
change `selectTxHistory`